src/libraries/System.Reflection.Metadata/src/System/Reflection/Internal/Utilities/DecimalUtilities.cs

```diff
@@ -7,7 +7,7 @@ internal static class DecimalUtilities
     {
         public static int GetScale(this decimal value)
         {
-#if NETCOREAPP
+#if NETCOREAPP_5_0_OR_GREATER
```

Comment:
Why these changes? This feels like more clutter, for no value.

Reply:
Same answer as in another question. Before I compiled this library against netcoreapp3.1 as well but undid the changes after some feedback from @ericstj. I can address this if I need to re-spin CI, otherwise I would revert this in a follow-up change.